x/mint/README.md

```diff
@@ -47,6 +46,68 @@ oneYear := time.Hour * 24 * 365
 | 19   | 1.50              |
 | 20   | 1.50              |
 
+## State
+
+See [./types/minter.go](./types/minter.go) for the `Minter` struct which contains all of the state of this module. The
+
+## State Transitions
+
+The `Minter` struct is updated every block via `BeginBlocker`.
+
+## Messages
+
+N/A
+
+## Begin Block
+
+See `BeginBlocker` in [./abci.go](./abci.go).
+
+## End Block
+
+N/A
+
+## Hooks
+
+N/A
+
+## Events
+
+An event is emitted every time a block provision is minted. See `mintBlockProvision` in [./abci.go](./abci.go).
+
+## Client
+
+### CLI
+
+```shell
+$ celestia-appd query mint annual-provisions
+80235005639941.760000000000000000
+```
+
+```shell
+$ celestia-appd query mint genesis-time
+2023-05-09 00:56:15.59304 +0000 UTC
+
+```
+
+```shell
+$ celestia-appd query mint inflation
+0.080000000000000000
+```
```

Comment:
TBH not sure this is valuable and pretty concerned it will become stale

Reply:
I would still leave the queries here

Reply:
sounds good will do